webapp/certified/views.py

```diff
@@ -125,6 +127,8 @@ def certified_model_details(canonical_id):
         "objects"
     ]
 
+    models = api.certified_models(canonical_id=canonical_id)["objects"]
+
```

Comment:
I would move this above the  component summaries call and add a 404 check. It is unlikely to be an empty list since there was a result on certified model details, but technically it's possible.

If it does 404, then you save an extra step of calling the component summaries api call when you didn't need to.

Reply:
Done, thanks